tests/network/vmi_multus.go

```diff
@@ -576,6 +580,45 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 			})
 		})
 
+		Context("kubectl get vmi test for primary and secondary interfaces", func() {
+			It("should verify vmi ip value match primary interface ip value", func() {
+				Expect(createBridgeNetworkAttachmentDefinition(util.NamespaceTestDefault, "linux-br", bridge10CNIType, bridge10Name, 0, "", bridge10MacSpoofCheck)).To(Succeed())
+
+				vmi := libvmi.NewCirros(libvmi.WithNetwork(v1.DefaultPodNetwork()))
+				vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{*newVMISpecIfaceWithBridgeBinding("nic1"), *v1.DefaultBridgeNetworkInterface(), *newVMISpecIfaceWithBridgeBinding("nic2")}
+				vmi.Spec.Networks = append(vmi.Spec.Networks, *newVMISpecMultusNetwork("nic1", "linux-br"), *newVMISpecMultusNetwork("nic2", "linux-br"))
```

Comment:
Please use the building options from `libvmi` to build this spec. It should be part of the `libvmi.NewCirros` input.

Reply:
fixed